test/axios2.test.js

```diff
@@ -1271,4 +1271,102 @@ describe("Axios AuthHttpRequest class tests", function() {
+                    logs.push(logText.split("$")[1]);
+                }
+            });
+            await page.evaluate(async () => {
+                let BASE_URL = "http://localhost.org:8080";
+                supertokens.init({
+                    apiDomain: BASE_URL,
+                    onHandleEvent: ev => console.log(`TEST_EV$${ev.action}`)
+                });
+                supertokens.addAxiosInterceptors(axios);
+                let userId = "testing-supertokens-website";
+
+                await axios.post(`${BASE_URL}/login`, JSON.stringify({ userId }), {
+                    headers: {
+                        Accept: "application/json",
+                        "Content-Type": "application/json"
+                    }
+                });
+                console.log("TEST_EV$LOGIN_FINISH");
+                await axios.post(`${BASE_URL}/update-jwt`, {
+                    headers: {
+                        Accept: "application/json",
+                        "Content-Type": "application/json"
+                    },
+                    body: JSON.stringify({ test: 1 })
+                });
+                console.log("TEST_EV$UPDATE1_FINISH");
+                await delay(5);
+                await axios.get(`${BASE_URL}/`, {
+                    headers: {
+                        Accept: "application/json",
+                        "Content-Type": "application/json"
+                    }
+                });
+                console.log("TEST_EV$REFRESH_FINISH");
+
+                await axios.post(`${BASE_URL}/update-jwt`, {
+                    headers: {
+                        Accept: "application/json",
+                        "Content-Type": "application/json"
+                    },
+                    body: JSON.stringify({ test: 2 })
+                });
+                console.log("TEST_EV$UPDATE2_FINISH");
+
+                await delay(5);
+                await axios.post(`${BASE_URL}/update-jwt`, {
+                    headers: {
+                        Accept: "application/json",
+                        "Content-Type": "application/json"
+                    },
+                    body: JSON.stringify({ test: 3 })
+                });
+                console.log("TEST_EV$UPDATE3_FINISH");
+
+                await axios.post(`${BASE_URL}/logout`, {
+                    method: "post",
+                    headers: {
+                        Accept: "application/json",
+                        "Content-Type": "application/json"
+                    },
+                    body: JSON.stringify({ userId })
+                });
+            });
+            assert.deepEqual(logs, [
+                "SESSION_CREATED",
+                "LOGIN_FINISH",
+                "ACCESS_TOKEN_PAYLOAD_UPDATED",
+                "UPDATE1_FINISH",
+                "REFRESH_SESSION",
+                "ACCESS_TOKEN_PAYLOAD_UPDATED",
+                "REFRESH_FINISH",
```

Comment:
Shouldn't this not have the `ACCESS_TOKEN_PAYLOAD_UPDATED` event?

Reply:
I later renamed this event to `ACCESS_TOKEN_UPDATED` since that is more accurate, and the next call after a refresh does update the access token.

Reply:
> and the next call after a refresh does update the access token.

I don't understand this. You removed the `ACCESS_TOKEN_PAYLOAD_UPDATED`, therefore the older `ACCESS_TOKEN_PAYLOAD_UPDATED` was incorrect?